EnoEngine/EnoEngine.CTF.cs

```diff
@@ -95,8 +98,12 @@ public async Task<DateTime> HandleRoundEnd(long roundId, bool recalculating = fa
             var json = JsonSerializer.Serialize(scoreboard, EnoCoreUtil.CamelCaseEnumConverterOptions);
             File.WriteAllText($"{EnoCoreUtil.DataDirectory}scoreboard{roundId}.json", json);
             File.WriteAllText($"{EnoCoreUtil.DataDirectory}scoreboard.json", json);
+
             jsonStopWatch.Stop();
             this.logger.LogInformation($"Scoreboard Generation Took {jsonStopWatch.ElapsedMilliseconds} ms");
+            var content = new StringContent(json, Encoding.UTF8, "application/json");
+            var response = await Client.PostAsync("http://localhost:5000/api/scoreboardinfo/scoreboard?adminSecret=secret", content);

```

Comment:
can we make this configurable somehow? since the plan is that the engine fetches the `ctf.json` from the landing page anyways, we would somehow configure the landing page URL for this as well

Reply:
for sure! I've already added it to the description but did not have time yesterday